### DISCUSSION TITLE
Any way to run the software on Linux?

### DISCUSSION THREAD

**Discussion by fschmid** [SafeExamBrowser/seb-win-refactoring#646] [General]
Hi 

As of 2023 is there any way to get this software (client) running on Linux? Anyone did it?

I found in the forum an tried 

https://github.com/eqsoft/seb2 

https://github.com/eqsoft/seb

but these projects are very old and doesn't seem to work anymore (in one case I get error messages about a profile error in the other case seb2 the target browser FF ESR 52 isn't available for download since a long time ago. (If you think too that these projects are non functional anymore please state so in the description, or delete them, it might safe valuable time from others.)

It's really annoying for me because your software is unfortunately the only one in my kids school software stack which doesn't seem to be functional on Linux (and Android) and therefore forces my school kids to use Windows or iOS. I don't believe this should be the case in an educational and openminded environment but thats only my opinion.

So if there is a way please give some advice thank you.

**Comment by danschlet** [comment]
On which Linux you want to use SEB? Fedora? Ubuntu? Debian? I don't think a universal version would be possible...

You can search our GitHub project for the term "linux", see https://github.com/search?q=repo%3ASafeExamBrowser%2Fseb-win-refactoring%20%2Flinux

https://github.com/SafeExamBrowser/seb-win-refactoring/issues/91
https://github.com/SafeExamBrowser/seb-win-refactoring/issues/44
https://github.com/SafeExamBrowser/seb-win-refactoring/issues/96

Unfortunately nothing changed since those posts. If no one offers to finance development of a linux version, there won't be any. 

Same with Android (and Google until today doesn't provide a single app/lockdown/kiosk mode for BYOD on Android, which makes it much harder to provide a lockdown browser than on iOS). For Android there's a commercial solution which is partially compatible with SEB: [Fully Kiosk Browser](https://www.fully-kiosk.com).

**Comment by danschlet** [comment]
PS: About the outdated https://github.com/eqsoft repositories: Create an issue there, these are not run by us.
